src/common/serviceInstanceHelpers.js

```diff
@@ -60,7 +60,8 @@ const DEFAULT_SERVICES = {
   THREESCALE: '3scale',
   APICURIO: 'apicurio',
   FUSE_MANAGED: 'fuse-managed',
-  RHSSO: 'rhsso'
+  RHSSO: 'rhsso',
+  MDC: 'Mobile Developer Console'
```

Comment:
Not sure what is the key and where it comes from. I do not see rhsso in config map so it is probably created somewhere else?

Reply:
How I can add mdc to it and use URL from the secret?